src/apps/lwaftr/icmp.lua

```diff
@@ -93,6 +94,22 @@ function new_icmpv4_packet(from_eth, to_eth, from_ip, to_ip, initial_pkt, l2_siz
    return new_pkt
 end
 
+function is_icmpv4(l2_size, pkt)
+   assert(l2_size > 2)
+   return rd16(pkt.data + l2_size - 2) == constants.ethertype_ipv4
```

Comment:
Constants.o_ethernet_ethertype
Easy to grep for, more readable, changeable if we ever back away from the current approach, less magic-number-y.